nodes/number/float_math.py

```diff
@@ -51,6 +56,9 @@ def outputIntegerChanged(self, context):
     outputInteger = BoolProperty(name = "Output Integer", default = False,
         update = outputIntegerChanged)
 
+    labelOperation = StringProperty(name = "labelOperation", default = "A * B", update = operationChanged)
+    labelOutputType = StringProperty(name = "labelOutputType", default = " (float)", update = outputIntegerChanged)
```

Comment:
the name attribute should be good for the ui, it doesn't have to look like a variable.  "Operation Label" would be better